.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: Please describe the behavior you expected
+    validations:
+      required: true
+  - type: textarea
```

Comment:
can you include images in textareas as well?

Reply:
Yeah it does, just added a tip that you can drag files into it